src/clj/game/core/hosting.clj

```diff
@@ -71,7 +71,7 @@
 
        ;; events should be registered for: runner cards that are installed; corp cards that are Operations, or are installed and rezzed
        (when (or (is-type? target "Operation")
-                 (is-type? target "Event")
+                 (and (is-type? target "Event") (false? facedown))
```

Comment:
For future-proofing, shouldn't the `(not facedown)` requirement apply to both sides?